ROADMAP.md

```diff
@@ -11,7 +11,7 @@ Welcome to the Cosmos SDK's team roadmap.
  
 * [Produce a spec for the new store design](https://github.com/cosmos/cosmos-sdk/issues/12986)
     * Research a new design for store. This could entail writing some POC's in order to identify design patterns
-* Store as its own go module
+* Store as its own go module **(Done)**
     * Store module should be its own go.mod without a dependency on the Cosmos SDK
 * Begin implementation of store v2
```

Comment:
You can reference the ADR if you'd like -> https://github.com/cosmos/cosmos-sdk/pull/15028